eahub/profiles/models.py

```diff
@@ -463,8 +464,10 @@ def is_searchable(self) -> bool:
     def get_absolute_url(self):
         return urls.reverse("profile", args=[self.slug])
 
-    def get_email_searchable(self) -> Optional[str]:
-        return self.user.email if self.email_visible else None
+    def messaging_url_if_allowed(self) -> str:
+        if self.allow_messaging:
+            return f'/profile/{self.slug}/message'
```

Comment:
@mattg95, we will need to replace it with a `reverse`, could you help here?

sadly hardcoded URLs stop silently working in the future